### ISSUE TITLE
README mistakes

### ISSUE THREAD

**Issue by lettleprince** [nicklockwood/SwiftFormat#204]
There are two mistakes in the content of README.

```
For a multiline comment, mark linebreaks with \n: `--header "First line\nSecond line"
```

should be

```
For a multiline comment, mark linebreaks with \n: --header "First line\nSecond line"
```

And,

```
`--header "Copyright (c) {year} Foobar Industries"`
```

should be

```
--header "Copyright (c) {year} Foobar Industries"
```

**Comment by nicklockwood** [comment]
Thanks. Could you open a PR to fix them, and I'll merge it?

**Comment by nicklockwood** [comment]
Never mind, fixed in 0.32.2
